src/models/__tests__/profile.test.ts

```diff
@@ -112,6 +112,50 @@ describe("findLastVersionByModelId", () => {
     }
   );
 
+  it.each`
+    case                                                              | pushNotificationsContentType | expectedPushNotificationsContentType
+    ${"existing profile with pushNotificationsContentType empty"}     | ${undefined}                 | ${"UNSET"}
```

Comment:
Add `UNSET` -> `UNSET` scenario

Reply:
done in https://github.com/pagopa/io-functions-commons/pull/297/commits/25f802f12c34ec047393044f870ae527936bc452